samples/SocketsSample/Hubs/Streaming.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Threading.Tasks.Channels;
+using Microsoft.AspNetCore.SignalR;
+
+namespace SocketsSample.Hubs
+{
+    public class Streaming : Hub
+    {
+        [return: Streaming]
```

Comment:
I originally had that thought but when we talked last we decided to use an attribute to make it clear. It's a little less clear around something like `IAsyncEnumerable<T>`. Do you want to load up all the data before returning a result or stream it out?